src/test/java/core/basesyntax/strategy/impl/PurchaseOperationTest.java

```diff
@@ -0,0 +1,46 @@
+package core.basesyntax.strategy.impl;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.strategy.OperationsStrategy;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class PurchaseOperationTest {
+    private static final String FRUIT_NAME = "banana";
+    private static final int BALANCE_FRUIT_QUANTITY = 105;
+    private static final int FRUIT_QUANTITY = 85;
+    private static final int INVALID_FRUIT_QUANTITY = 120;
+    private static OperationsStrategy purchaseOperation;
+
+    @BeforeAll
+    static void beforeAll() {
+        purchaseOperation = new PurchaseOperation();
+    }
+
+    @BeforeEach
+    void setUp() {
+        Storage.storage.clear();
+        Storage.storage.put(FRUIT_NAME, BALANCE_FRUIT_QUANTITY);
+    }
```

Comment:
You don't need to put something in storage in setUp, better put data in tests, where it's necessary